openapi/dcos.yaml

```diff
@@ -3132,27 +3119,34 @@ components:
             will be used if present.
           nullable: true
         cpus:
+          default: 0.9
```

Comment:
Let me know if I am wrong, but shouldn't we default on the service-defaults instead of setting them here ?

Reply:
No idea, to be honest. I just moved the value around.

I think there is/was some confusion in the original spec regarding the default values: at some places the default seems to be used as an "example" value (which OpenAPIv3 supports in a separate "example" field) and it some places it's used as "this is what comes down from the server". There's a slight difference.

Reply:
We want to rely on the defaults of the edgeLB service instead of having them here. What if these defaults change in the service when these properties are not specified ? We shouldn't set these defaults

Reply:
I think these defaults only have an effect on the documentation to describe what the server would use if nothing is specified.

Changing/adding defaults does not change the code, as far as I can tell, and thus not the requests being made. See the generated code.

Does that make sense? Otherwise I can still remove them?

Reply:
As discussed in our zoom call, let's remove these defaults cause it'll increase the maintenance load.